React/Base/RCTUIKit.h

```diff
@@ -172,6 +172,9 @@ UIKIT_STATIC_INLINE CGFloat UIFontLineHeight(UIFont *font)
 // UIInterface.h/NSUserInterfaceLayout.h
 #define UIUserInterfaceLayoutDirection NSUserInterfaceLayoutDirection
 
+// UIAlertController.h/NSViewController.h
+#define UIAlertController NSViewController
```

Comment:
Running iOS RNTester revealed RCTAlertController class specified wrong base class, as I wanted to make it compile on macOS. Decided to go with the convention and redefine this symbol

Reply:
As I understood it, this file is meant for the more generic symbols. This definition can probably just go in `RCTAlertController.h`? /cc @HeyImChris @tom-un

Reply:
I'll do that, makes more sense as this is the only usage

Reply:
Should be good to go now